src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt

```diff
@@ -274,6 +274,13 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
         }
     }
 
+    private fun formatKeyAndName(key: String, name: String): String {
+        val safeKey = key ?: ""
+        val formattedName = if(name != null && name != "") "($name)" else ""
```

Comment:
you were right before that in Kotlin has type safety so `key` and `name` can't be `null`. I think it's only necessary to ensure that if `name` is an empty string, we don't show the brackets. We shouldn't do the extra checks that the language enforces for us.